docs/bokeh/source/docs/first_steps/first_steps_1.rst

```diff
@@ -91,14 +100,20 @@ Follow these steps to recreate this simple line chart:
         # add a line renderer with legend and line thickness to the plot
         p.line(x, y, legend_label="Temp.", line_width=2)
 
-5. Finally, use the |show| function to generate your graph and
+6. Finally, use the |show| function to generate your graph and
    open a web browser to display the generated HTML file.
 
     .. code-block:: python
 
         # show the results
         show(p)
 
+7. From the command line, run the Python script you just created. For example:
+
+    .. code-block:: python
```

Comment:
I think we want `none` here (or possibly `sh` for shell) since this is not, itself, python code
```suggestion
    .. code-block:: none
```

Reply:
Yes, this should be `.. code-block:: sh`!